pkg/logcli/index/volume.go

```diff
@@ -45,9 +47,9 @@ func (q *VolumeQuery) volume(rangeQuery bool, c client.Client) *loghttp.QueryRes
 	var err error
 
 	if rangeQuery {
-		resp, err = c.GetVolumeRange(q.QueryString, q.Start, q.End, q.Step, q.Limit, q.Quiet)
+		resp, err = c.GetVolumeRange(q.QueryString, q.Start, q.End, q.Step, q.Limit, q.TargetLabels, q.AggregateByLabels, q.Quiet)
```

Comment:
Would it be worth just passing the query to these so we don't have to change all the interfaces when we want to add stuff